Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -0,0 +1,380 @@
+.. include:: ../../Includes.txt
+
+.. _flexforms:
+
+==============
+FlexForms
+==============
+
+FlexForms can be used to configure extension plugins in the backend.
+
+Credits
+=======
+
+Some of the examples were taken from the extensions
+`news <https://extensions.typo3.org/extension/example/>`__ (by Georg Ringer)
+and `bootstrap_package <https://extensions.typo3.org/extension/example/>`__
+(by Benjamin Kott). This chapter was originally written by Sybille Peters.
+
+Further enhancements by the TYPO3 community are welcome!
+
+When to Use FlexForms
+=====================
+
+Extensions can be configured using :ref:`Extension Configuration <extension-options>`.
```

Comment:
I would not talk to much about Extension Configuration. If this changes at some time, we need to check this section, and this will likely be forgotten. I would keep the section and only focus on FlexForms without comparisons.

Maybe we should also extend this section, or add another one, "When NOT to use FlexForms".